administrator/language/de-DE/plg_system_schemaorg.ini

```
@@ -0,0 +1,22 @@
+; Joomla! Project
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2023 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_SYSTEM_SCHEMAORG="System – Schema.org"
+PLG_SYSTEM_SCHEMAORG_BASETYPE_DESCRIPTION="Auswählen, ob die Website eine Organisation oder eine einzelne Person repräsentiert."
+PLG_SYSTEM_SCHEMAORG_BASETYPE_LABEL="Basis Typ"
+PLG_SYSTEM_SCHEMAORG_BASETYPE_OPTION_ORGANIZATION="Organisation"
+PLG_SYSTEM_SCHEMAORG_BASETYPE_OPTION_PERSON="Person"
+PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_DESCRIPTION="Strukturierte Daten sind ein standardisiertes Format für die Organisation und Darstellung von Informationen im Internet. Diese bietet die Möglichkeit, den Inhalt und die Bedeutung von Daten auf strukturierte Weise zu beschreiben, um es Suchmaschinen und anderen Anwendungen erleichtert, die Informationen zu verstehen und zu verarbeiten. <a href='https://schema.org' target='_blank' rel='noopener noreferrer'>Weitere Informationen auf schema.org</a>."
+PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_DESCRIPTION_NOT_CONFIGURATED="Um die schema.org-Funktionalität zu nutzen, muss zunächst das Plugin durch den Administrator der Seite konfiguriert werden."
+PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_DESCRIPTION_NOT_CONFIGURATED_ADMIN="Um die schema.org-Funktionalität zu nutzen, muss zunächst das Plugin konfiguriert werden. Auf <a href='index.php?option=com_plugins&amp;task=plugin.edit&amp;extension_id=%s' target='_blank'>diesen Link klicken um das Plugin zu öffnen</a>, konfigurieren und zu speichern."
+PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_LABEL="Schema"
+PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_TYPE_LABEL="Schema Typ"
+PLG_SYSTEM_SCHEMAORG_IMAGE_LABEL="Bild"
+PLG_SYSTEM_SCHEMAORG_NAME_LABEL="Name"
+PLG_SYSTEM_SCHEMAORG_SOCIALMEDIA_LABEL="Social Media Konten"
```

```suggestion
PLG_SYSTEM_SCHEMAORG_SOCIALMEDIA_LABEL="Social Media Profile"
```

In dem Kontext sind Profile auf Social Media Plattformen gemeint.